<ui version="4.0" >
 <class>UserConfiguration</class>
 <widget class="QWidget" name="UserConfiguration" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Bitstream Vera Sans'; font-size:8pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">WARNING:&lt;/span> All changes made here will be &lt;span style=" font-weight:600;">applied&lt;/span> instantly.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Database:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlComboRequester" name="urDatabaseUrl" />
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tvUser" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KPushButton" name="pbAdd" >
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbChangePassword" >
       <property name="text" >
        <string>Change Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KPushButton" name="pbDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlComboRequester</class>
   <extends>KUrlRequester</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
